spec/api-debugger-spec.ts

```diff
@@ -29,13 +29,9 @@ describe('debugger module', () => {
       expect(w.webContents.debugger.isAttached()).to.be.true();
     });
 
-    it('fails when protocol version is not supported', done => {
-      try {
-        w.webContents.debugger.attach('2.0');
-      } catch {
-        expect(w.webContents.debugger.isAttached()).to.be.false();
-        done();
-      }
+    it('fails when protocol version is not supported', async () => {
```

Comment:
`w.webContents.debugger.attach('2.0')` throws, not rejects - I don't think there's any reason to have `async` here?

Reply:
I have mirrored the existing
https://github.com/electron/electron/blob/7981d955b8f84e2129f5a907445fbc0aea6b9fe8/spec/api-debugger-spec.ts#L41-L44

Reply:
fixing both